mm2src/mm2_main/tests/docker_tests/docker_tests_common.rs

```diff
@@ -1608,7 +1608,7 @@ pub fn init_geth_node() {
             SEPOLIA_ETOMIC_MAKER_NFT_SWAP_V2 =
                 EthAddress::from_str("0x9eb88cd58605d8fb9b14652d6152727f7e95fb4d").unwrap();
             SEPOLIA_ERC20_CONTRACT = EthAddress::from_str("0xF7b5F8E8555EF7A743f24D3E974E23A3C6cB6638").unwrap();
-            SEPOLIA_TAKER_SWAP_V2 = EthAddress::from_str("0x7Cc9F2c1c3B797D09B9d1CCd7FDcD2539a4b3874").unwrap();
+            SEPOLIA_TAKER_SWAP_V2 = EthAddress::from_str("0x3B19873b81a6B426c8B2323955215F7e89CfF33F").unwrap();
```

Comment:
And this too

Reply:
post merge conflicts

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2323#pullrequestreview-2663371395